Roadmap.md

```diff
@@ -1,188 +1,235 @@
+Contributors are encouraged to focus their efforts on work that aligns with the roadmap then we can work together in these areas.
 
-## Remove blockers to Meteor adoption
+PRs to the roadmap are welcome. If you are willing to contribute please open a PR explaining your ideas and what you would be able to do yourself.
 
-### Make the `meteor` command-line tool installable from npm
+Ideally, every item in this roadmap should have at least two leaders, leaders are people that are interested in the item and would like to help. If you are interested please open a PR including yourself and describing how do you want to help.
 
-Installing `meteor` from npm would enable developers to use it as build tool for npm-based projects, and would simplify the Meteor release process by getting rid of the "dev bundle" (essentially the npm dependencies of the command-line tool).
+## Core
+### Update to Node.js 12
+- Leaders: [Ben Newman](https://github.com/benjamn)
+- Status: In Progress
+- PRs: https://github.com/meteor/meteor/pull/10527
 
-The biggest blockers to this project are
+Since Node.js 12 is scheduled to become the LTS version on October 1st, 2019, Meteor 1.9 will update the Node.js version used by Meteor from 8.16.1 (in Meteor 1.8.2) to 12.10.0 (the most recent current version).
 
-1. deciding whether/how to preserve Meteor release versions, and
-2. changing the API of the package server so that you don't have to download the entire package database locally.
+### Tree Shaking
+- Leaders: [Ben Newman](https://github.com/benjamn)
+- Status: -
+- PRs: -
 
-## Page load performance improvements
+Implement tree shaking / dead code elimination, which involves pruning the dependency tree while scanning imports in the `ImportScanner`. We believe it should be possible to treat values like `Meteor.isProduction` as constants during this process, and eliminate those branches if their conditions are false, so that the `./routes` dependency would not show up at all in the production bundle.
 
-*Ongoing*
+### Service worker build target
+- Leaders: <you?>
+- Status: -
+- PRs: -
 
-Speeding up page load times will require a combination of new tools for asynchronous JavaScript delivery (code splitting), dead code elimination, deferred evaluation of JavaScript modules, and performance profiling (so that developers can identify expensive packages).
+A proper service worker build target. Regular Web Workers can be built from a function.toString() but service-workers require an actual server route.
 
-### Making large dependencies optional
+### Ultra-thin Meteor
+- Leaders: <you?>
+- Status: -
+- PRs: -
 
-Making it possible to remove (or dynamically load) large dependencies like `jquery`, `underscore`, and `minimongo` will have a significant impact on bundle sizes.
+Have a ultra-thin version of Meteor that does not depend in anyway to mongo. Allow completely shutting down DDP and dependencies on DDP.
```

Comment:
I have looked into this few months ago and in fact it works pretty well, the only issue is that hot-reload currently uses DDP: https://github.com/meteor/meteor/issues/9960 I think that should be moved to its own socket and 

I opened few related issues:
* https://github.com/meteor/meteor-feature-requests/issues/31
* https://github.com/meteor/meteor-feature-requests/issues/354
* https://github.com/meteor/meteor/issues/10701
* https://github.com/meteor/meteor/issues/10702
* https://github.com/meteor/meteor/issues/10704

Reply:
Thanks @mitar , I just added these links to the description.

Are you willing to be one of the leaders in this item?